controlplane/kubeadm/internal/controllers/helpers_test.go

```diff
@@ -261,6 +272,7 @@ func TestReconcileKubeconfigSecretDoesNotAdoptsUserSecrets(t *testing.T) {
 	g.Expect(r.Client.Get(ctx, secretName, kubeconfigSecret)).To(Succeed())
 	g.Expect(kubeconfigSecret.Labels).To(Equal(existingKubeconfigSecret.Labels))
 	g.Expect(kubeconfigSecret.Data).To(Equal(existingKubeconfigSecret.Data))
+	//TODO: (killianmuldoon) find out how to achieve this with consistent adoption.
```

Comment:
Q: The test seems to behave as expected now, right? (the ownerRef is not added because it's a user-provided secret) And it's doing that consistently.

Is there anything to do here?